cpp/__test_server.cpp

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
please move this to `tests/test_server.cpp` (along with the Tupfile)

Reply:
no, I argue for the tupfile to be able to build the project to your needs. Let me explain my *vision* using lightd: lightd has a tupfile in the top level directory. There you include fibre as a subdirectory and tup will automatically build the fibre and use it in the ligthd project without the need to specify every fibre file necessary for this. I'm currently refactoring the tupfile and will remove it from the PR for the time being.